kubeyard/entrypoints/kubeyard.py

```diff
@@ -173,6 +177,10 @@ def deploy(**kwargs):
 
 @cli.command(help=DebugCommand.__doc__)
 @apply_common_options(initialized_repository_options)
+@click.argument(
+    'name',
+    required=False
```

Comment:
```suggestion
    required=False,
    type=str,
```

Reply:
What's the convention regarding explicitly setting args equal to default?
I tried to follow are uses of `@click.option` which omit the `type` unless it is different from default.

Reply:
`type` is only proposal (due to lack of help message) but you need to add comma, then :+1:

Reply:
oh, comma, right. Added :)